history-of-quakers/original/20-chapter-18.adoc

```diff
@@ -846,7 +880,7 @@ with several of his friends that accompanied him.
 
 Now whilst I leave him on ship-board, I cannot forbear to mention,
 that this year at London came forth a witty pamphlet with this title,
-An easy Way to get Money cum Privilegio, without fear or cumber,
+[.book-title]#An Easy Way to Get Money Cum Privilegio, Without Fear or Cumber,#
```

Comment:
I've had to google this a few times -- I think when title-casing a book, the modern convention is to leave prepositions lowercased, unless they are the start of the Title.  so here `without` should be lowercase, and apparently the latin word `cum` means "with" so let's put that back to lowercase as well. Full disclosure: I had to google whether _without_ was a preposition. 😊

But you were right in uppercasing Easy, Get, Fear, Cumber. 👍

Reply:
Haha, yeah, I had actually put it into a "title generator" thing where it capitalizes titles for you so I take no responsibility for this! 😆 I'm learning Spanish and have been looking at so much English grammar again and it's been really interesting to review it all-- and yes, I'm not sure if I would have known "without" was a preposition either. :)